packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -77,6 +77,23 @@ export interface BaseServiceOptions {
    * @default - AWS Cloud Map service discovery is not enabled.
    */
   readonly cloudMapOptions?: CloudMapOptions;
+
+  /**
+   * Specifies whether to propagate the tags from the task definition or the service to the tasks in the service
+   *
+   * Valid values are: PropagateTagFromType.SERVICE or PropagateTagFromType.TASK_DEFINITION
+   *
+   * @default PropagatedTagSource.NONE
```

Comment:
You say default here is none, but the logic below is not none

Reply:
Done!